jane/doc/parallelism-tutorial.md

```diff
@@ -1,38 +1,162 @@
-# Parallelism Tutorial
+# Simple Parallelism Without Data Races
 
 # Introduction
 
-# Fork/join parallelism
+OCaml 5 unleashed parallel programming for the OCaml ecosystem. As is often the
+case when something is unleashed, this has quite a bit of potential to cause
+chaos. **[Need to quit fussing with this so I'm leaving it as is for the moment,
+but am I saying we're putting the leash back on? Leashes are good, actually? Or
+maybe we're putting up a fence so the the multicore dog can run around without
+data races biting people??]**
 
-Suppose we're working with binary trees, and we want to take an average over all
-the values in the tree. Here's a basic implementation:
+OxCaml aims to provide an infrastructure for multicore OCaml programming that
+exploits the full power of parallelism while _guaranteeing_ freedom from an
+especially pernicious threat, that of the _data race._ That infrastructure
+includes many parts, from common primitives like atomics and locks to newer
+features like modes and capsules. Fortunately, not every parallel program will
+need all of this machinery, and following this tutorial, you'll see how far you
+can get with just two modes and the `Parallel.fork_join2` function.
+
+## What is a data race?
+
+A data race is a race condition in which two accesses to the same memory
+location conflict. Specifically, there are four elements of the crime: **[[Oops,
+I forgot to define “domain” first]]**
```

Comment:
Yes, maybe write a little section before this one entitle "A very brief introduction to multicore OCaml" or something